addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -109,7 +109,7 @@ const Wysiwyg = Widget.extend({
             getYoutubeVideoElement: getYoutubeVideoElement,
             getContextFromParentRect: options.getContextFromParentRect,
             getPowerboxElement: () => {
-                const selection = (this.options.document || document).getSelection();
+                const selection = this.odooEditor.document.getSelection();
```

Comment:
The bug this commit is trying to fix was fixed here: https://github.com/odoo/odoo/pull/85304
And what you're doing here is guaranteed not to work since you're calling `this.odooEditor` before defining it.
Anyway not needed since the powerbox works just fine in mass mailing.

Reply:
You fixed the same issue in the meantime.
I wouldn't consider this code black magic.
I'm removing the commit however because it doesn't add much.